vcpkg/commands/new.md

```diff
@@ -0,0 +1,146 @@
+---
+title: vcpkg new
+description: Command line reference for the vcpkg new command. Creates a new manifest in the current directory.
+ms.author: bion
+ms.date: 07/05/2023
+ms.prod: vcpkg
+---
+# vcpkg new
+
+## Synopsis
+
+```console
+vcpkg new --application
+```
+
+```console
+vcpkg new --name hello --version 1.0
+```
+
+Creates a manifest file ([`vcpkg.json`](../reference/vcpkg-json.md)) and a default configuration file ([`vcpkg-configuration.json`](../reference/vcpkg-configuration-json.md)) in the current working directory.
+Creates a manifest file (`vcpkg.json`) and default configuration file (`vcpkg-configuration.json`)
+ready to use in your project.
+
+Use the `--application` flag if your project is an end-user application.
+
+If not using `--application`, `--name` and `--version` are required. `--version-date`,
+`--version-relaxed`, or `--version-string` can force that a particular version format is selected.
+For more information about the versioning formats, see
+[Versioning reference](../users/versioning.md).
+
+## Options
+
+All vcpkg commands support a set of [common options](common-options.md).
+
+### `--application`
+
+Creates a manifest suitable for use in applications, removing the requirement to supply name and
+version.
+
+### `--name`
+
+The name to write into the manifest.
+
+### `--version`
+
+Indicates the version to write into the manifest. If none of the other version format switches are
+passed, infers the appropriate form to use based on the form of the input.
+
+### `--version-relaxed`
+
+Indicates that the version to write into the manifest is a 'relaxed' version. Relaxed versions
+obey semantic versioning's format, but allow any number of numeric values.
+
+### `--version-date`
+
+Indicates that the version to write into the manifest is a 'date' version. Date versions are an
+ISO 8601 date.

```

Comment:
Maybe add a link?

Reply:
ISO standards aren't publicly accessible. Or do you want a link to a wikipedia page or something like that?

Reply:
I think we can just add an example of the format and link back to our versioning reference documentation for more information.